plugin-server/src/worker/ingestion/PoE.md

```diff
@@ -0,0 +1,107 @@
+# Persons on Events
+
+## Background
+
+We're getting ready to make a substantial change to the way [persons](https://posthog.com/docs/data/persons) and [events](https://posthog.com/docs/data/events) work by essentially combining them and adding person ids and properties onto events. This is the way we’ll be querying data for all teams using PostHog in the near future.
+
+Why are we doing this? Firstly, it makes queries significantly faster because we no longer have to join tables to get a result; we can just look up everything in the events table instead. One query in our internal tests showed a 400x increase in speed, though 10x is the most common figure...but this beta will help us understand the improvement in real-world conditions.
+
+Secondly, feedback showed that users weren't able to create queries based on person properties at the time of an event. By putting person properties on events, this becomes the new default, while still enabling you to filter insights based on the latest properties using cohorts.
+
+## Roll-out
+
+We're currently offering the new query experience for Beta testers. As it is in Beta, you may or may not spot a few bugs here and there, please use the in-app bug reporting option and mention that you're a PoE (Persons on Events) beta tester.
+
+Since this isn't fully ready yet we might offer you one of the roll-out stage options from below:
+
+## None: JOIN based queries
+
+As reference here are the two aspects of queries that might change and how they work before PoE:
+
+### 1. Filtering on person properties
+
+Latest person properties are joined in during query time.
+
+### 2. Insights counting unique persons
+
+Distinct_id to Person mapping is joined together at query time, see [docs](https://posthog.com/docs/how-posthog-works/queries#insights-counting-unique-persons).
+
+## PoEv1
+
+### 1. Filtering on person properties
+
+Properties from ingestion time are used, see [docs](https://posthog.com/docs/how-posthog-works/queries#filtering-on-person-properties)
```

Comment:
This "ingestion time" stuff is a bit confusing

Reply:
sth like this:
```suggestion
Person properties at the time the event was processed are used, see [docs](https://posthog.com/docs/how-posthog-works/queries#filtering-on-person-properties)
```